tailwind.config.js

```diff
@@ -2,7 +2,16 @@
 module.exports = {
   content: ["./src/**/*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        webGreen: "#00672E",
+        webGrey: "#EFECEA",
```

Comment:
have meaningful names when adding custom tailwind utility classes
```suggestion
     /* Note: before adding a color in tailwind environment, first check if the colors are not in tailwind env already */ 
        'green-250': "#00672E",
        'gray-250': "#EFECEA",
```